src/model/keychain.js

```diff
@@ -67,3 +74,27 @@ export async function remove(key) {
     captureException(err);
   }
 }
+
+export async function loadAllKeys(authenticationPrompt) {
+  try {
+    const { results } = await getAllInternetCredentials(authenticationPrompt);
+    return results;
+  } catch (err) {
+    logger.log(`Keychain: failed to loadAllKeys error: ${err}`);
+    captureException(err);
+  }
+  return null;
+}
+
+export async function getAllKeysAnonymized() {
+  const data = {};
+  const results = await loadAllKeys();
+  results.forEach(async result => {
+    data[result.username] = {
```

Comment:
probs overkill given that result if it exists SHOULD have username and password but could we put ? in here everywhere :)

Reply:
I think it's safe https://github.com/rainbow-me/rainbow/pull/965/files#diff-99837e2b9d78f3c88a65d464c4c4b757R42 but I'll add it anyways